test/code-highlighter.html

```diff
@@ -85,15 +85,33 @@
       it('tapping on last line should not fire', function (done) {
         elem.code = 'one liner \n'
         var icon = Polymer.dom(elem.root).querySelector('iron-icon');
-        flush(function () {
-          elem.addEventListener('add-hunk', function () {
-            done(new Error('This event should not have been fired'));
-          });
+        elem.addEventListener('add-hunk', function () {
+          done(new Error('This event should not have been fired'));
         });
         MockInteractions.tap(icon);
         setTimeout(done, 10);
```

Comment:
10 millisec timeout is not really enough. Slow builds will essentially disable this test because the timeout fires before any event gets dispatched. I'd suggest using 100 ms instead.